### PR TITLE
Added PySpark3 kernel.

### PR DESCRIPTION
**Pull request by tc0312**
image

### CODE REVIEW

File: sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```
Comment (aggFTW) [inline sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py]: This doesn't need a version number, does it?

Comment (msftristew) [inline sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py]: No

Comment (tc0312) [inline sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py]: I will remove the version.
I cloned this from pysparkkernel. I can remove versions in pysparkkernel and sparkkernel in a different PR if you want.

**Comment by linchan-ms** [comment]
LGTM

File: sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py (hunk 1 above)
Comment (aggFTW) [inline sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py]: Mmm... don't know why that was there in the first place, but thanks! That would be good!

**Comment by aggFTW** [comment]
Looks great after the changes! Merge after Travis passes 👍

**Comment by tc0312** [comment]
Mind if we do a squash merge to hide the feedback commit? BTW I don't have right to push changes to sparkmagics.

**Pull request merged**